test/unittest/conftest.py

```diff
@@ -1,113 +1,52 @@
-import json
 import random
-from test.material.metrics import PatK
+import uuid
+from pathlib import Path
 
-import lorem
-import numpy
 import pytest
 
 try:
-    from test.material.models import BinaryClassifier
-
     import torch
 
     from superduperdb.ext.torch.encoder import tensor
     from superduperdb.ext.torch.model import TorchModel
 except ImportError:
     torch = None
 
+from superduperdb import CFG
 from superduperdb.backends.mongodb.query import Collection
+from superduperdb.base.build import build_datalayer
 from superduperdb.base.datalayer import Datalayer
 from superduperdb.base.document import Document
 from superduperdb.components.dataset import Dataset
 from superduperdb.components.listener import Listener
-from superduperdb.components.metric import Metric
 from superduperdb.components.vector_index import VectorIndex
-from superduperdb.ext.numpy.encoder import array
 from superduperdb.ext.pillow.encoder import pil_image
 
 n_data_points = 250
```

Comment:
Let's stick with one convention: lower or upper case for constants (I'm not bothered either way).
In the past I've seen global constants with upper case.

Reply:
Yes, Let use upper case for constants, WDYT? @thejumpman2323

Reply:
I  have changed the constants to uppercase.

Reply:
sure